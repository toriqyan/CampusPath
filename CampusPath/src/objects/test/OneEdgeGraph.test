# Tests that a graph can be created, a pair of nodes and
# an edge between them can be added, and a pair of nodes
# and a single edge can be printed out.

# Create a graph
CreateGraph TheLordOfTheRing1

# Add a pair of nodes
AddNode TheLordOfTheRing1 Bilbo
AddNode TheLordOfTheRing1 Gandalf

# Add an edge
AddEdge TheLordOfTheRing1 Bilbo Gandalf TheHobbit

# Print the nodes in the graph and the outgoing edges from Bilbo
ListNodes TheLordOfTheRing1
ListChildren TheLordOfTheRing1 Bilbo