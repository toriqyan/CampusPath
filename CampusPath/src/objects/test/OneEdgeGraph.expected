# Tests that a graph can be created, a pair of nodes and
# an edge between them can be added, and a pair of nodes
# and a single edge can be printed out.

# Create a graph
created graph TheLordOfTheRing1

# Add a pair of nodes
added node Bilbo to TheLordOfTheRing1
added node Gandalf to TheLordOfTheRing1

# Add an edge
added edge TheHobbit from Bilbo to Gandalf in TheLordOfTheRing1

# Print the nodes in the graph and the outgoing edges from Bilbo
TheLordOfTheRing1 contains: Bilbo Gandalf
the children of Bilbo in TheLordOfTheRing1 are: Gandalf(TheHobbit)